kit/wifi.md

```diff
@@ -6,69 +6,59 @@ title: WiFi
 WiFi
 ====
 
-The WiFi dongles provided with your kit allow you to interface with and debug your robot.
+The Raspberry Pi that makes up your kit's brain board contains a WiFi radio which allows you to interface with and debug your robot.
 You can connect to your robot using any WiFi capable device (laptop, tablet, phone, etc.)
 
 Connecting to Your Robot
 ------------------------
 
-First, ensure that the WiFi dongles supplied with your kit are plugged into your robot.
-Once you have the WiFi dongles connected, plug in a USB memory stick and turn your robot on.
-
-During the boot process you should notice a green LED begin to flash on one of the WiFi dongles.
-Your robot has now set up its very own WiFi network! It'll be called `robot-XXX`, where `XXX` is
-your team's TLA.
+During the boot process you should notice the green LED labelled WiFi turn on.
+Your robot has now set up its very own WiFi network! It will have a name starting with `robot-ZZZ` followed by some random numbers.
 
 You can now connect to your robot in the same way you normally connect to a WiFi network.
```

Comment:
Presumably you mean when the robot is in `COMP` mode rather than merely at the competition?
If we can let the interface be read-only in `COMP` mode I think that's valid (though if it can't be then that's probably preferable to allowing it to essentially be a remote control).

Reply:
480171c

Reply:
Disabling the WiFi is separate to comp mode. The comp mode USB sticks do 5 things:
- set comp mode
- set zone
- set game timeout
- set marker offset
- disable WiFi

They are largely independent and setting comp mode won't automatically do any of the others.

For here though the generalisation is fine for the moment.